Makefile.include

```diff
@@ -478,6 +478,10 @@ reset:
 	$(call check_cmd,$(RESET),Reset program)
 	$(RESET) $(RESET_FLAGS)
 
+TESTS ?= $(wildcard $(APPDIR)/tests/*)
```

Comment:
can we make this a bit more specific, i.e., match on python scripts - just in case there are other files or subdirs that are not supposed to be run